src/cargo.rs

```diff
@@ -38,10 +38,10 @@ fn cargo(project: &Project) -> Command {
     let mut cmd = raw_cargo();
     cmd.current_dir(&project.dir);
     cmd.envs(cargo_target_dir(project));
-    cmd.envs(rustflags::envs());
+    cmd.env_remove("RUSTFLAGS");
```

Comment:
I believe this break our usage.

We pass deny warnings using the RUSTFLAGS environment variable.
This is very handy to ensure no warnings ever gets emitted from our macros.

```rust
#[test]
fn tests() {
	std::env::set_var("RUSTFLAGS", "--deny warnings");
	let t = trybuild::TestCases::new();
	t.compile_fail("tests/*rs");
}
```

Is there any way to keep rustflags here ? or maybe allow to pass TRYBUILD_RUSTFLAGS variable and use it here?

If you approve the direction I can make a PR